### ISSUE TITLE
RESOLUTION: We will ask the accessibility people to look at IMEs and the problems we should have considered here but didn’t.

### ISSUE THREAD

**Issue by johanneswilm** [w3c/editing#83]

**Comment by johanneswilm** [comment]
People have been contacted and made aware.
